### ISSUE TITLE
AMD Ryzen 5 5600G

### ISSUE THREAD

**Issue by casudo** [isc30/ryzen-7000-series-proxmox#24]
Does anyone know if it works with the AMD Ryzen 5 5600G?

**Comment by Riddle1988** [comment]
Yes, it worked for me, but I extracted a vbios from my motherboard bios zip file. I tested it only with Windows 10 VM and Proxmox 7.1 and 8.2.
You can find a lot of usful info on unraid forum:
https://forums.unraid.net/topic/112649-amd-apu-ryzen-5700g-igpu-passthrough-on-692/page/6/

For me RadeonBugFix with reboot is not working and I would like to try it with WIndows 11 one day and maybe Linux somehow.



![image](https://github.com/isc30/ryzen-7000-series-proxmox/assets/22612114/91d596cc-0925-41bd-b3c7-e6714775977b)
